crc/services/study_service.py

```diff
@@ -121,14 +122,14 @@ def get_study_associate(study_id = None, uid=None):
             raise ApiError('uid not specified','A valid uva uid is required for this function')
 
         if uid == study.user_uid:
-            return {'uid': ownerid, 'role': 'owner', 'send_email': True, 'access': True}
+            return {'uid': uid, 'role': 'owner', 'send_email': True, 'access': True}
```

Comment:
Here is where we should use an object or a schema.  Seems like the full model should be returned, then passed through the schema.